<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVariableDialog</class>
 <widget class="QDialog" name="AddVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Variable</string>
  </property>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DestinationComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OriginLabel">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Origin</string>
   </property>
  </widget>
  <widget class="QLabel" name="DescriptionLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="EquationLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Equation</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddVariableLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Add Variable</string>
   </property>
  </widget>
  <widget class="QLabel" name="DestinationLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>210</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DescriptionLE">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EquationLE">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="OriginComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>EquationLE</tabstop>
  <tabstop>DescriptionLE</tabstop>
  <tabstop>OriginComboBox</tabstop>
  <tabstop>DestinationComboBox</tabstop>
  <tabstop>AddButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
